CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe Installer will be documented in this file.
 
+## `3.3.1`
+
+- Bugfix: z/OSMF static definition is missing scheme information, this is needed for AT-TLS hybrid scenarios where connections TO z/OSMF do not follow AT-TLS rules [#4512](https://github.com/zowe/zowe-install-packaging/pull/4512)
```

Comment:
`where connections TO z/OSMF do not follow`
Is `TO` typo or does it mean something?

Reply:
looks like a typo or for emphasis, I lowercased it